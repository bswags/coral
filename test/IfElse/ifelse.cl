#cordbconn
server="server";
user="server";
password="server";
port="server";
DBName="server";
type="server";
#enddbconn

#cordb
#enddb

void main() {

	if(1 == 1){
		printf("1 is equal to 1\n");
	}
	if(2 > 1){
		printf("2 is greater than 1\n");
	}
	if(1 < 2){
		printf("1 is less than 2\n");
	}


	if(1 != 1){
		printf("nope\n");
	}
	else{
		printf("1 is not equal to 1");
	}
    
}