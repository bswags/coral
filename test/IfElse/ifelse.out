Traceback (most recent call last):
  File "IfElse/ifelse.clx", line 3, in <module>
    import coral_backend
ImportError: No module named coral_backend
